common/app/views/fragments/email/footer.scala.html

```diff
@@ -1,23 +1,9 @@
 @()
 
-@import views.support.Zurb._
-
-@fullRow(Seq("social__links")) {
-    <a class="social__link" href="https://www.facebook.com/theguardian">
-        @fragments.inlineSvg("fb", "icon") 5.4m
-    </a>
-
-    <a class="social__link" href="https://twitter.com/guardian">
-        @fragments.inlineSvg("twitter-bird", "icon") 5.2m
-    </a>
-
-    <a class="social__link" href="https://www.instagram.com/guardian">
-        @fragments.inlineSvg("instagram", "icon") 272k
-    </a>
-}
+@import views.support.EmailHelpers._
 
 @fullRow(Seq("email__links")) {
-    <a class="email__link" href="">Manage your emails</a> | <a class="email__link" href="">Unsubscribe</a>
+    <a class="email__link" href="https://profile.theguardian.com/email-prefs">Manage your emails</a> | <a class="email__link" href="http://click.mail.theguardian.com/?qs=699722b7e50d4eb4c88091e8d2b9b341b0bba21aed1e528c64a42b5605713f23">Unsubscribe</a>
```

Comment:
What's the query string there? Should it be hardcoded or is it some kind of identifier that should get inserted at send time?